image_width: 1920
image_height: 1200
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [4853.60021,    0.     , 1194.54233,
            0.     , 4852.61557,  393.90298,
            0.     ,    0.     ,    1.     ]
camera_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.305940, -2.492739, -0.021472, 0.013820, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [4890.59717,    0.     , 1201.0196 ,    0.     ,
            0.     , 4888.84717,  387.55318,    0.     ,
            0.     ,    0.     ,    1.     ,    0.     ]
